ranger/ext/img_display.py

```diff
@@ -17,12 +17,13 @@
 import fcntl
 import imghdr
 import os
+import shlex
 import struct
 import sys
 import warnings
 import json
 import threading
-from subprocess import Popen, PIPE
+from subprocess import Popen, PIPE, check_output
```

Comment:
Can we get `check_output` from `ranger.ext.spawn`? Then we don't need to give up Python 2.6 compatibility.

Reply:
Yup!